common/src/main/java/org/apache/syncope/common/services/ReportService.java

```diff
@@ -192,4 +195,25 @@
     @Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
     Response exportExecutionResult(@NotNull @PathParam("executionId") Long executionId,
             @QueryParam("format") ReportExecExportFormat fmt);
+    
+    
+    /**
+     * List all the reports of a particular type
```

Comment:
Please change this description: it should be something like as "List report jobs of the given type"